### PR TITLE
added files

### PR DESCRIPTION
**Pull request by shmielminz**
Hope it's all good please review

### CODE REVIEW

File: 3_data.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+use GownDB
+go
+delete dbo.Gown
+go
+insert dbo.Gown(CustomerFirstName,CustomerLastName,Brand,Style,Color,Size,GownNumber,DateBorrowed,DateReturned,ConditionBorrowedInt,ConditionReturnedInt)
+select 'Sharon', 'Weiss', 'Pullini', 'puffy', 'lavendar', 6, 2, '9/17/21', '9/25/21',4,4
+union select 'Sharon', 'Weiss','Pullini', 'puffy', 'lavendar', 6, 1, '9/17/21', '9/25/21', 3,2
+union select 'Chayala', 'Halpert', 'Shar tulle', 'pale', 'pink', 8, 2, '8/10/21', '8/27/21', 3,3
```
Comment (Shevawedeck) [inline 3_data.sql]: pale pink is the color and tulle is the style

File: 3_data.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+union select 'Chayala', 'Halpert', 'Shar tulle', 'pale', 'pink', 6, 1, '8/10/21', '8/27/21', 3,2
+union select 'Chayala', 'Halpert', 'Pullini puffy', 'pale', 'pink', 4, 3, '8/10/21', '8/27/21', 2,2
+union select 'Chayala', 'Halpert', 'Pullini puffy', 'pale', 'pink', 2, 2, '8/10/21', '8/27/21',2,1
```
Comment (Shevawedeck) [inline 3_data.sql]: pale pink is the color and puffy is the style

**Review by Shevawedeck — CHANGES_REQUESTED** [review]

**Comment by Shevawedeck** [comment]
there are a few changes needed

**Comment by shmielminz** [comment]
Sorry, I have minimal idea on colors and gown styles. Don't know all those female styles and colors. 😜

**Pull request closed without merging**